crates/bevy_window/src/event.rs

```diff
@@ -290,3 +292,18 @@ pub struct WindowMoved {
     /// Where the window moved to in physical pixels.
     pub position: IVec2,
 }
+
+/// An event that is sent when system changed window theme.
+///
+/// It is only sent when [`Window::preferred_theme`](crate::window::Window::preferred_theme) is `None`.
```

Comment:
This means that events will only spawn when deferring to the system's default theme? Could you add that bit to clarify this here too?

Reply:
Right, the event is only sent when the window is relying on the system theme to control its appearance and the system theme has changed. Pushed a PR to give more detail to the documentation.

Reply:
Thanks! Left a ✅ on that PR, looks good.